Makefile

```diff
@@ -334,6 +334,11 @@ eks-test-packages: ## eks test packages
 aks-test-packages: ## aks test packages
 	@./control-plane/build-support/scripts/set_test_package_matrix.sh "acceptance/ci-inputs/aks_acceptance_test_packages.yaml"
 
+
+.PHONY: openshift-test-packages
+openshift-test-packages: ## openshift test packages
+	@./control-plane/build-support/scripts/set_test_package_matrix.sh "acceptance/ci-inputs/openshift_acceptance_test_packages.yaml"
+
```

Comment:
Is this only for CI or could I run make openshift-test-packages locally?

Reply:
You could run this locally if you were connected to an openshift cluster and passed in the --use-openshift flag